packages/build-tools/src/builders/ios.ts

```diff
@@ -164,6 +165,9 @@ function resolveArtifactPath(ctx: BuildContext<Ios.Job>): string {
   if (ctx.job.artifactPath) {
     return ctx.job.artifactPath;
   } else if (ctx.job.simulator) {
+    if (isTVOS(ctx, ctx.job.buildConfiguration)) {
+      return 'ios/build/Build/Products/*-appletvsimulator/*.app';
+    }
     return 'ios/build/Build/Products/*-iphonesimulator/*.app';
```

Comment:
```suggestion
    return 'ios/build/Build/Products/*simulator/*.app';
```
not sure if it's possible to make a build where both directories exists, but in that case we probably want to upload everything, plus it simplifies logic

remember to update docs https://docs.expo.dev/build-reference/eas-json/#artifactpath-1

Reply:
I'm not a big fan of 
```
if (sth) {
  return 1;
}
return 2;
```
IMO this pattern should be used only for throwing errors, returning early from function, etc. In this case, the code below `if` is basically the `else` branch. It reads better if you do
```
if (sth) {
  return 1;
} else {
  return 2;
}
```
because from the first look you know this is an alternative and the `if` is not a "special" case.